travvy-server/app.js

```diff
@@ -39,3 +39,8 @@ app.post('/login',(req,res) =>{
     message: 'Successful log in'
   })
 })
+app.get('/getUser',(req,res) =>{
+  res.send({
+    message: 'Got user'
```

Comment:
Nice confirmation message to let the system know that the users profile has been successfully obtained from the users table